bridge/client/app/_views/ktb-settings-view/ktb-settings-view.component.scss

```diff
@@ -0,0 +1,3 @@
+::ng-deep .settings-section {
```

Comment:
As it will probably be used in more than one component from now on, you could move it to `styles.scss` to omit the `::ng-deep` selector

Reply:
I would leave it here because this component is the parent for all settings